Sources/WordPressUI/Extensions/Gravatar/GravatarURL+Util.swift

```diff
@@ -13,7 +13,7 @@ extension GravatarURL {
     public static func url(for email: String,
                            preferredSize: ImageSize? = nil,
                            gravatarRating: GravatarRating? = nil,
-                           defaultImageOption: DefaultImageOption? = nil) -> URL? {
```

Comment:
I missed this before, they've been passing 404 by default. So I added it to our method as well.